### DISCUSSION TITLE
Camel Cased Content Components breaking in Nuxt3.9.1

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#25172] [Questions]
I was on nuxt `3.8.2` and everything worked fine, and when I upgraded to nuxt `3.9.1` I noticed that any camel cased component in the content directory would break. 

E.g. in `components/content/FooBar.vue` now since I am using nuxt 3.9.1 `<foobar></foobar>` will be outputted, the only way I can make this work is if I don‘t use any camel casing `components/content/Foobar.vue`

link to the reproduction https://stackblitz.com/edit/nuxt-starter-snapag?file=content%2Fblog%2Ftest.md

**Comment by TheAlexLichter** [comment]
Your repro work as expected 🤔 

![image](https://github.com/nuxt/nuxt/assets/640208/d7a0b5dd-49c8-46ee-900d-465479d66a36)

**Reply by dbzx10299** [reply]
In my repo the component is named `Foobar.vue` try changing it to `FooBar.vue` (camel cased) and it will not work.

**Reply by dbzx10299** [reply]
This only started since I upgraded nuxt from 3.8.2 to 3.9.1

**Comment by TheAlexLichter** [comment]
Tracking in https://github.com/nuxt/nuxt/issues/25166
